src/repository/sigobj.rs

```diff
@@ -426,22 +461,32 @@ impl SignedAttrs {
                     else if !strict {
                         cons.skip_all()
                     } else {
-                        xerr!(Err(decode::Malformed.into()))
+                        Err(cons.content_err(InvalidSignedAttr::new(oid)))
                     }
                 })? { }
                 Ok(())
             })
         })?;
         if raw.len() > 0xFFFF {
-            return Err(decode::Unimplemented.into())
+            return Err(cons.content_err(
+                "signed attributes over 65536 bytes not supported"
```

Comment:
s/65536/65535/

Reply:
Technically, the message isn’t wrong ;)

(Anyway: fixed.)